CHANGELOG.rst

```diff
@@ -35,6 +35,8 @@ Improved Documentation
   xdist could identify the wrong test as failing when test crashes at teardown.
   To address this issue, xdist now marks test as complete at teardown.
 
+- 168.trivial: add long option `--numprocesses` as alternative for `-n`.
```

Comment:
So I remove it from the CHANGELOG file? Sorry if I'm a bit confused I just haven't worked with fragment files and tool build changelog files before